test/test.c

```diff
@@ -11,20 +11,19 @@ int main(void)
 {
 	int ret = 0;
 	Logger_SetLevel(LOGGER_LEVEL_OFF);
-	ret += test_charset();
-	ret += test_linkedlist();
-	ret += test_string();
-	ret += test_strpool();
-	ret += test_object();
-	ret += test_thread();
-	ret += test_font_load();
-	ret += test_image_reader();
-	ret += test_xml_parser();
-	ret += test_widget_event();
-	ret += test_widget_opacity();
-	ret += test_textview_resize();
-	ret += test_textedit();
-	PRINT_TEST_RESULT(ret);
+	describe("test charset", test_charset);
+	describe("test linkedlist", test_linkedlist);
+	describe("test string", test_string);
+	describe("test strpool", test_strpool);
+	describe("test object", test_object);
+	describe("test thread", test_thread);
+	describe("test font_load", test_font_load);
+	describe("test image_reader", test_image_reader);

```

Comment:
Underscores should be replaced with spaces

```diff
- font_load
+ font load
```

Reply:
Done. Verified all describe() calls.